integration/core/test_api.py

```diff
@@ -11,3 +12,34 @@ def client():
 def test_catalog_list(client):
     templates = client.list_template()
     assert len(templates) > 0
+
+
+def test_templatebase_eq_filter(client):
+    url = 'http://localhost:8088/v1-catalog/templates?templateBase_eq=invalid'
```

Comment:
Can you add a test somewhere that asserts the new `templateBase` field exists and matches what is expected?